### ISSUE TITLE
Accessibility - Map contrast issue

### ISSUE THREAD

**Issue by chmnata** [CityofToronto/bdit_cot-vfh#58]
Issue
The colours in the map to distinguish between the different percentage does not have enough contrast.

The colour #FFFEC9 (colour for 0.2) against the background colour of #FFFFFF has a contrast ratio of 1.03:1 out of a required contrast ratio of at least 3:1 for graphics.

The colour #FFEDA0 (colour for 1.9) against the background colour of #FFFFFF has a contrast ratio of 1.17:1 out of a required contrast ratio of at least 3:1 for graphics.

The colour #FED676 (colour for 3.6) against the background colour of #FFFFFF has a contrast ratio of 1.39:1 out of a required contrast ratio of at least 3:1 for graphics.

The colour #FEB250 (colour for 5.3) against the background colour of #FFFFFF has a contrast ratio of 1.79:1 out of a required contrast ratio of at least 3:1 for graphics.

The colour #FD893C (colour for 7.0) against the background colour of #FFFFFF has a contrast ratio of 2.38:1 out of a required contrast ratio of at least 3:1 for graphics.

The colour #FFEDA0 (colour for 1.9) against the colour of #FD893C (colour for 7.0) has a contrast ratio of 2.29:1 out of a required contrast ratio of at least 3:1 for graphics.

Recommendation
WCAG 2.0 level AA requires a contrast ratio of at least 4.5:1 for normal text and 3:1 for large text. WCAG 2.1 requires a contrast ratio of at least 3:1 for graphics

**Comment by chmnata** [comment]
Changed colours to hexarray: ["#ca6194", "#ffbe6e", "#fa8775", "#340074", "#b22399"]
no colour deficiency
![image](https://user-images.githubusercontent.com/46324452/150002096-8a3e4122-b51c-46ae-abe3-ce68870ab98c.png)
Deuteranomaly
![image](https://user-images.githubusercontent.com/46324452/150002158-041c2bc3-08a7-437f-add4-6bf594e4486d.png)
Protanomaly
![image](https://user-images.githubusercontent.com/46324452/150002181-7d7b2e72-1b15-4061-bc6f-dd24fcb0b33a.png)
Protanopia
![image](https://user-images.githubusercontent.com/46324452/150002198-0cf2fc80-eb20-4037-b3ed-aaff58d40ec2.png)
Deuteranopia
![image](https://user-images.githubusercontent.com/46324452/150002216-4b4ba6bf-8e1a-449c-8ea4-ae92c0ece88a.png)
Grey Scale
![image](https://user-images.githubusercontent.com/46324452/150002236-0a7e323a-0e52-4a75-b4eb-925fc29f1d08.png)

**Comment by chmnata** [comment]
However, the island with null value is in grey and will not have enough contrast with other colours. I will add pattern to null values.

**Comment by chmnata** [comment]
Ended up tweaking the colours a little and used white for null value since the border colour was changed to a dark grey (1:9.59)
![image](https://user-images.githubusercontent.com/46324452/150013116-7e39fcf4-d5e3-414b-81c8-c733c7507ae2.png)

**Comment by chmnata** [comment]
![image](https://user-images.githubusercontent.com/46324452/150013369-b9562180-02bc-4b0f-b368-8044a5546813.png)
![image](https://user-images.githubusercontent.com/46324452/150013391-d45654e9-391c-4ebc-b514-4eede95f455e.png)
![image](https://user-images.githubusercontent.com/46324452/150013414-d63b2b0b-83a6-42df-9539-abc3cd38edc7.png)
![image](https://user-images.githubusercontent.com/46324452/150013440-c03b1d41-5e24-4cc0-8a89-d8ad3fe75ae7.png)
![image](https://user-images.githubusercontent.com/46324452/150013457-26c321c9-13b3-4b8c-b979-995f1242d503.png)
![image](https://user-images.githubusercontent.com/46324452/150013475-0016f3ae-f015-46cc-a1df-466de2ac4e7d.png)
